File: year2020/month12/day1222/ms_1222.cpp
#include <iostream>
#include <string>
#include <sstream>
#include <vector>

using namespace std;

double solution(vector<int> arr);

int main() {
	vector<int> arr;
	string input;

	getline(cin, input);
	stringstream ss(input);

	while (getline(ss, input, ' ')) {
		int temp = stoi(input);

		if (-10000 <= temp && temp <= 10000) {
			arr.push_back(temp);
		}
	}

	if (1 <= arr.size() && arr.size() <= 100) {
		cout << solution(arr) << endl;
	}

	return 0;
}

double solution(vector<int> arr) {
	double answer = 0;

	for (int i = 0; i < arr.size(); i++) {
		answer += arr[i];
	}

	answer /= arr.size();

	return answer;
}

File: year2020/month12/day1222/ms_1222.cs
using System;

namespace algorithm_127
{
    class ms_1222
    {
        static double solution(int[] arr) 
        {
            double answer = 0;

            foreach(var value in arr)
            {
                answer += value;
            }

            answer /= arr.Length;

            return answer;
        }

        static void Main(string[] args)
        {
            String[] input = Console.ReadLine().Split(' ');
            int[] arr = new int[0];
            int len = 0;

            for(int i = 0; i < input.Length; i++)
            {
                int data = Convert.ToInt32(input[i]);

                if(-10000 <= data && data <= 10000)
                {
                    if(arr.Length == len)
                    {
                        int[] temp = new int[len + 1];

                        for(int j = 0; j < len; j++)
                            temp[j] = arr[j];

                        arr = temp;
                    }

                    arr[len] = data;
                    len++;
                }
            }

            if(1 <= len && len <= 100)
            {
                Console.WriteLine(solution(arr));
            }
        }
    }
}
